panda/src/audio/config_audio.cxx

```diff
@@ -75,6 +75,13 @@ ConfigVariableInt audio_preload_threshold
           "practical to stream multiple sound-files from disk at the same "
           "time - the hard drive seek time makes it stutter."));
 
+ConfigVariableBool audio_want_hrtf
+("audio-want-hrtf", true,
+ PRC_DESC("This determines whether OpenAL-Soft should activate HRTF in "
+          "certain hardware configurations. Set it to true to cause "
+          "OpenAL to automatically apply HRTF processing to all output "
+          "audio when headphones are used for playback."));
```

Comment:
Should we make this an openal-specific config variable?

Reply:
There are almost no config variables that specify that they are for openal by name, most of the config variables that do in fact only apply to openal just start contain "audio" (I know this for a fact with `audio-preload-threshold` and I'm sure there are many others).

So my way making it an "openal-specific config" was just to place it along side the other ones. Let me know if there's more I should do for that.